app/views/artisans/SolutionProblemsView.coffee

```diff
@@ -0,0 +1,193 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/artisans/solutionProblemsView'
+Level = require 'models/Level'
+Campaign = require 'models/Campaign'
+Level = require 'models/Level'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class SolutionProblemsView extends RootView
+  template: template
+  id: 'solution-problems-view'
+  excludedCampaigns = [
+    "picoctf"
+    "auditions"
+
+    #"dungeon"
+    #"forest"
+    #"desert"
+    #"mountain"
+    #"glacier"
+
+    "dungeon-branching-test"
+    "forest-branching-test"
+    "desert-branching-test"
+
+    #"intro"
+    #"course-2"
+    #"course-3"
+    #"course-4"
+    #"course-5"
+    #"course-6"
+  ]
+  excludedSimulationLevels = [
+    # Course Arenas
+    "wakka-maul"
+    "cross-bones"
+  ]
+  excludedSolutionLevels = [
+    # Multiplayer Levels
+    "cavern-survival"
+
+    "dueling-grounds"
+    "multiplayer-treasure-grove"
+
+    "harrowland"
+
+    "zero-sum"
+
+    "ace-of-coders"
+    "capture-their-flag"
+  ]
+  levelOffset: 0
+  isReady: 0
+  requiresSubs: 0
+  rob: []
+  constructor: (options) ->
+    super options
+    @campaigns = new CocoCollection([], 
+      url: '/db/campaign?project=slug'
+      model: Campaign
+    )
+    @campaigns.fetch()
+    @listenTo(@campaigns, 'sync', @onCampaignsLoaded)
+    @supermodel.loadCollection(@campaigns, 'campaigns')
```

Comment:
Same here, `initialize` instead of `constructor`, `Campaigns` instead of `CocoCollection`, etc.